nss-day-cohort-39/tabloid-full-stack-bobs
# CC-Deactivate User Profile

**CHANGES**
- Added a UserProfile component to the client
- Added feature to user profiles that allows admins to deactivate accounts.

**TESTING**
- Run the application from VS and VSCode.
- Login as an ADMIN USER
- From the home page, click the User Profiles tab.
- Click the deactivate button on a user and then verify that the profile is successfully removed from the user profile list. 
- Check your UserProfile data table in Visual Studio to ensure that the user has a FALSE value under the IsActive column.
- Log out and now log in again as an AUTHOR USER.
- Return to the User Profiles list and ensure that the "Deactivate" and "Edit" buttons to not appear.
